### DISCUSSION TITLE
How to center navbar dropdown menus?

### DISCUSSION THREAD

**Discussion by currentcreative** [holger1411/understrap#2147] [Q&A] [answered]
I'd like to have the dropdown menus in the navbar centered under their respective top links. I've added the Bootstrap 5 class "dropdown-center" class everywhere I can through webdev tools and nothing works.

**Comment by IanDelMar** [comment]
Can you please share a screenshot of the markup? And share a link to a plain HTML version where `dropdown-center` works in a navbar?

**Reply by IanDelMar** [reply, marked as answer]
Obviously, you've already found the answer that it doesn't work (https://github.com/twbs/bootstrap/issues/36897)...

**Reply by currentcreative** [reply]
Thanks for the response. Yeah, I found that after searching a bit more. Seems crazy! I'm surprised there aren't a lot more people who want to do this. I have a feeling this client is going to probably request the dropdowns be centered as soon as they see it.
